### PR TITLE
Implemented bookstore app styling

### PR DESCRIPTION
**Pull request by Shaheryar0054**
- Implement the CSS in order to give styling. 
- My final design matches the Zepline design.
- Added some elements in order to match the Zepline design.

### CODE REVIEW

File: src/components/Navbar.js [hunk 1]
```diff
@@ -9,19 +12,21 @@ import {
...
     <Router>
-      <h1>Bookstore CMS</h1>
-      <nav>
-        <ul>
+      <nav id="nav-container">
+        <h1>Bookstore CMS</h1>
+        <ul id="pages-container">
           <li>
-            <Link to="/">Books</Link>
+            <Link className="Books" style={{ textDecoration: 'none' }} to="/">Books</Link>
```
Comment (adel-gu) [inline src/components/Navbar.js]: - [ ] Please consider updating the books link to be displayed in capital letters as shown in the design system 👍🏼 

### Your design 
image

### Design System
image

File: src/styles/book.css [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+  padding-left: 20px;
+  padding-right: 20px;
+  letter-spacing: 1px;
+}
+
+.user-icon {
+  border: 1px solid black;
+  padding: 13px;
+  border-radius: 24rem;
+  color: #0290ff;
```
Comment (adel-gu) [inline src/styles/book.css:51]: -[ ] Kindly consider updating the border color of the account Icon to `#e8e8e8` which is given in the Design System 👏🏼 

### Your Design
Screenshot from 2023-05-20 13-59-40

### System Design
Screenshot from 2023-05-20 14-00-11

File: src/styles/book.css [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 0.4rem solid #0290ff;
+  border-radius: 50%;
+  width: 4rem;
+  height: 4rem;
+  background-color: #fff;
+  border-bottom-color: rgb(175, 175, 175);
+  transform: rotate(240deg);
```
Comment (adel-gu) [inline src/styles/book.css]: - [ ] Please consider fixing the rotation degree so the blue-cirlce will duplicate the exact same look as the one in the System Design, try `rotate(135deg)` 💪🏼 

### Your Design
Screenshot from 2023-05-20 14-04-38

### System Design
Screenshot from 2023-05-20 14-05-24

File: src/styles/book.css [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+.Categories {
+  color: #a3a3a3;
+}
+
+#all-btn {
+  margin-left: -5px;
+  display: flex;
+  gap: 1rem;
+  align-items: center;
+}
```
Comment (adel-gu) [inline src/styles/book.css:69]: - [ ] The spacing between each button seems perfectly balanced but the separator `|` is distracting the spacing so here are my suggestions:

1. Remove the `border-right` styling from the class `all-btn .comments` and `#all-btn .remove` 👍🏼 
2. Add `<span>|</span>` between each button in HTML file 👍🏼 

### Your Design
Screenshot from 2023-05-20 14-14-51

### System Design
Screenshot from 2023-05-20 14-15-22

**Review by adel-gu — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES ♻️ 
**Hi @Shaheryar0054  👋🏼** 

Good job so far! 👏🏼👏🏼👏🏼 however, there are some issues that you still need to work on to go to the next project but you are almost there! 💪🏼💪🏼💪🏼. 



## Highlights
* No ID selectors for multiple elements ✔️ 
* No !important overusing ✔️ 
* No inefficient selectors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/book.css [hunk 5]
```diff
@@ -0,0 +1,218 @@
...
+  align-items: center;
+  border-bottom: 1px solid #e8e8e8;
+  background-color: white;
+}
+
+#nav-container h1 {
+  color: #0290ff;
+  font-size: 1.875rem;
+  margin-left: 5rem;
+}
```
Comment (shella12) [inline src/styles/book.css:33]: - [ ] [OPTIONAL] I would suggest you to adjust the size or position of the heading to make all the text fit in one line to match the given design perfectly.

|Your design| Given design|
|---|-----|
|Screenshot 2023-05-20 at 7 52 27 PM|Screenshot 2023-05-20 at 7 57 41 PM|

File: src/styles/book.css [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  color: white;
+  cursor: pointer;
+  font-size: 1rem;
+}
```
Comment (shella12) [inline src/styles/book.css]: - [ ] [OPTIONAL] I would suggest you to fit all input fields and button in one line to match the given design perfectly.

|Your design| Given design|
|---|-----|
|Screenshot 2023-05-20 at 7 52 17 PM|Screenshot 2023-05-20 at 7 59 34 PM|

**Review by shella12 — APPROVED** [review]
Hi @Shaheryar0054, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great design Implementation 👏 
- Correct color contrast 💯 
- Clean UI ✨ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**